src/send_data.py

```diff
@@ -0,0 +1,49 @@
+from ast import alias
+from xmlrpc.client import boolean
+from sensor_temp import getTemp
+from sensor_motion import getMotion 
+from alarm import alarm_on
+from button import button_pressed
+import requests
+import json
+import time
+
+
+URL = 'http://127.0.0.1:8000'
+
+## SEND DATA OUT
+
+valid_data = {}
```

Comment:
Is `valid_data` needs to be global variable?

Reply:
We removed this.